modules/Microsoft.ContainerRegistry/registries/deploy.bicep

```diff
@@ -170,6 +170,9 @@ param diagnosticEventHubName string = ''
 @description('Optional. The name of the diagnostic setting, if deployed.')
 param diagnosticSettingsName string = '${name}-diagnosticSettings'
 
+@description('Optional. Enables registry-wide pull from unauthenticated clients. It\'s in preview and available in the Standard and Premium service tiers.')
```

Comment:
Should this be tested as enabled in the common variant? Also, what should the default be?

Reply:
The default should be false, as it should pretty much always be for anonymous access.
I didn't add it to the test simply because it is a boolean with a value -  +so the only thing the test would do is to test if Azure works, not the template